python-threatexchange/threatexchange/exchanges/impl/stop_ncii_api.py

```diff
@@ -68,16 +68,16 @@ class StopNCIICredentials(auth.CredentialHelper):
     ENV_VARIABLE: t.ClassVar[str] = "TX_STOPNCII_KEYS"
     FILE_NAME: t.ClassVar[str] = "~/.tx_stopncii_keys"
 
-    function_key: str
     subscription_key: str
+    fetch_function_key: str
 
     @classmethod
     def _from_str(cls, s: str) -> "StopNCIICredentials":
-        user, _, passw = s.strip().partition(",")
-        return cls(user, passw)
+        subscription_key, _, fetch_function_key = s.strip().partition(",")
+        return cls(fetch_function_key, subscription_key)
```

Comment:
blocking: Hard to follow. The dataclass has subscription_key first and fetch_function_key second. I expect the constructor to have the same order.

Since you are using positional args, we're setting self.subscription_key to local variable fetch_function_key defined here.

Reply:
If your test plan is working, things may be correctly getting tagged, but I don't know why.. :P

Reply:
You got confused by the same thing I did. I can reverse back the order in the constructor as long as the order in the file is maintained.